spring-security-oauth2/src/main/java/org/springframework/security/oauth2/provider/error/DefaultWebResponseExceptionTranslator.java

```diff
@@ -111,8 +113,8 @@ public int getHttpErrorCode() {
 	@SuppressWarnings("serial")
 	private static class ServerErrorException extends OAuth2Exception {
 
-		public ServerErrorException(String msg, Throwable t) {
```

Comment:
No need to remove this constructor. Let's keep it. This needs to be a passive and binary compatible change.

Reply:
given that the class itself is private is it necessary for binary compatibility? but sure I can make that change

Reply:
also worth saying my constructor isn't public for any particular reason, just copying the code around it